docs/source/operating-daml.rst

```diff
@@ -2,9 +2,12 @@
 .. SPDX-License-Identifier: Apache-2.0
 
 Operating Daml
-===============
+==============
 
 .. toctree::
   :maxdepth: 2
 
-  ops/pruning
\ No newline at end of file
+  ops/connect/index
```

Comment:
This line has been removed on purpose in a previous PR. Please remove it.

Reply:
https://github.com/digital-asset/daml/pull/13052

Reply:
Removed.